templates/people.html

```diff
@@ -84,6 +84,9 @@ <h2>
                     <div class="people-role people-role-top-level people-role-sme">SMEs</div>
                     <div class="people-role-description-text">(Subject Matter Experts) have proven themselves to be experts in a given development area. They help guide Bevy's direction in their area and they are great people to reach out to if you have questions about a given area.</div>
                 </div>
+                <div class="people-role-description">
+                    <div class="people-role-description-text">The other members, without a special role, are part of the triage team. They can manage GitHub issues and pull requests.</div>
```

Comment:
```suggestion
                    <div class="people-role-description-text">Oher members, without a special role, are part of the GitHub organization. They have the power to manage GitHub issues and pull requests.</div>
```

Reply:
Why do you prefer the term "GitHub organization" over "triage team"? I'm not against it, I just want to know your thoughts.

Reply:
The triage team is an artifact at this point: we should try to remove it if we can.